src/pages/Dashboard/Portfolio/index.tsx

```diff
@@ -5,7 +5,7 @@ import { useTranslation } from 'react-i18next'
 import { ThemeContext } from 'styled-components'
 
 import { useActiveWeb3React } from 'src/hooks'
-import { AllChain, useGetChainsBalances } from 'src/state/portifolio/hooks'
```

Comment:
can we hold back on these changes, the reason is, we are moving portfolio to components repo https://github.com/pangolindex/components/pull/41 and removing Portfolio from https://github.com/pangolindex/interface/pull/519. so it will create unnecessary conflicts. what do you think ?

Reply:
alright, I have reverted typo fix commit.